### ISSUE TITLE
Weird color glitches

### ISSUE THREAD

**Issue by azteca1998** [Marc-Pierre-Barbier/Genshin-Impact-on-linux#3]
Hi, I've been playing GI for a while on Linux with "no problem" thanks to your guide.

But I'm having some trouble finding things in the world due to messed-up colors. Here's an example of what I'm talking about:
![imatge](https://user-images.githubusercontent.com/6751252/103175815-2982b880-486d-11eb-997e-ef1ace8fa3f4.png)

It's like a saturation bug where something causes at least one of the red, green and blue values to be maximum. Also there are reflections where there shouldn't be, [such as in paved roads in Monstadt](https://mega.nz/file/J5gGlQhR#O34iWMxcVmgl6VWjKRF-kBu_MsCekhXfYgz3rGW0P8c):

The game itself is playable, but really annoying to be searching for things and not seeing them right in front of you. For example, when trying to find aogate crystals... everything red that you can see on this picture is just rocks:
![imatge](https://user-images.githubusercontent.com/6751252/103175877-ad3ca500-486d-11eb-9d43-164eea502187.png)

I've followed your instructions step by step except the directx step as I didn't have any directx installer. Also the vmware tools should be the exact version, but I had to search for them since the link was broken when I installed them.

Trying to adjust graphics settings seems to change nothing. There was one time when the game started and ran as it was supposed to (first launch after the game updated to 1.3). I've also had some problems some time ago because the game was freezing the entire vm (had to power off the vm and restart it).

Specs:
  - AMD Ryzen 7 2700
  - 32GB RAM (16 available to the VM)
  - Radeon RX5700 (I've never had the black screen bug even though using an AMD graphics card)
  - Ubuntu 20.10

Finally, I've tried using discord at first but the lutris discord server is invalid, and I don't know how to send a DM to someone who isn't in my friends list on discord (either that or I'm blind).

Thanks.

**Comment by Marc-Pierre-Barbier** [comment]
this bug is linked to the vmware gpu the only way to fix it is to wait for vmware to fix it
you might want to try to contact the vmware support to complain about it.
the more complaint the faster they will consider to fix it

we updated the link to the lastest vmware tools releases you might want to try that

**Comment by azteca1998** [comment]
I understand. I was hoping for a setting or similar which tweaked the driver, but I guess there isn't (not with closed drivers at least).

I just tried installing the latest vmware tools, but that changed nothing... Do I need to uninstall the previous vmware tools first? Or does the installer do that automatically?

Guess I'll go bug the vmware support team a bit. However I don't think I'll be well received since I had to change `mks.gl.allowBlacklistedDrivers = "TRUE"` for 3d acceleration to work at all.

Since I'm not sure whether I should close this issue (since the problem is not solved, but can't be solved without vmware's intervention), I'll leave that to you.

Thanks.

**Comment by Marc-Pierre-Barbier** [comment]
wait what do you mean by closed source? are you using the closed source driver for you amd gpu instead of the open source ?

i never had one but i heard that the support was great using the opensource driver.

**Comment by azteca1998** [comment]
No, that one can't be installed on this version of ubuntu. I'm using the open source one (which comes preinstalled on ubuntu). What I mean by closed source is the vmware gpu driver, not the AMD one on the host. But now that I think about it, it could also mean the game itself.

**Comment by Marc-Pierre-Barbier** [comment]
ok thanks for the clarification
im not going to close this until someone report that it's fixed

**Comment by zhaodice** [comment]
I have the same problem,and I am running vmware 16.1 in windows 10 with NVIDIA GTX 1650,the game screen is same to that.

**Comment by zyrotheisz** [comment]
I had the same problem. In my case I didn't set the video ram to exact 2GB (it was a bit higher). Setting the vram to exact 2gb did it for me.

**Comment by smiley** [comment]
Just for documentation's sake (v1.3 doesn't load at all): [turning off fast startup](https://superuser.com/a/1347749) (aka hibernated kernel) fixed this for me, and the game looked _perfect_. Turning on v-sync also fixed some frame stuttering the game had originally, but that's a common issue even outside of VMs.

**Comment by billsboard** [comment]
For the record, Parallels Desktop has blinding colours no matter what I do. Also objects randomly disappear.
